src/plugins/intel_cpu/src/nodes/space_to_batch.cpp

```diff
@@ -112,6 +105,19 @@ static std::vector<size_t> getShape5D(const SizeVector &shape) {
 
 template<typename T>
 void SpaceToBatch::SpaceToBatchKernel() {
+    const auto *blockShapesPtr = reinterpret_cast<int *>(getParentEdgeAt(1)->getMemoryPtr()->GetPtr());
```

Comment:
There is no gurantee graph will produce I32 precision on this input. To ahieve that you have explicitly specify the precision during supportedPrimitive initialization stage (addSupportedPrimDesc): `{LayoutType::ncsp, Precision::I32},`

Reply:
Done